config/application.rb

```diff
@@ -64,6 +64,8 @@ class Application < Rails::Application
     # Ensure the correct user scope is used for authentication
     config.middleware.use BopsCore::Middleware::User, global_subdomains: %w[config]
 
+    config.exceptions_app = routes
```

Comment:
This is not a good idea - we've just had an exception and now we're sending the request back to the rack app that generated the exception. We should build a custom exceptions app if we really want to do this.